packages/components/src/DataTable/DataTable.css

```diff
@@ -100,3 +100,28 @@
   background: var(--color-surface--background);
   cursor: pointer;
 }
+
+.emptyState {
+  display: flex;
+  width: 100%;
+  text-align: center;
```

Comment:
Shouldn't these two lines be using

```
justify-content: center;
align-items: center;
```

if the goal is to always keep the empty state content centered?